packages/react-core/src/components/Page/Page.tsx

```diff
@@ -178,6 +184,12 @@ export class Page extends React.Component<PageProps, PageState> {
         tabIndex={mainTabIndex}
         aria-label={mainAriaLabel}
       >
+        {tertiaryNav && isTertiaryNavWidthLimited && (
+          <nav className={css(styles.pageMainNav, styles.modifiers.limitWidth)}>
```

Comment:
The page section should just be a `<div>` - the `<Nav>` react component that renders inside is an html `<nav>` element.

```suggestion
          <div className={css(styles.pageMainNav, styles.modifiers.limitWidth)}>
```